src/app/pages/home/Employees/modals/ModalEmployees.js

```diff
@@ -174,18 +174,16 @@ const ModalEmployees = ({
   });
 
   const executePolicies = (catalogueName) => {
-    const formatDate = new Date()
-    const dformat = `${('0' + formatDate.getDate()).slice(-2)}/${('0' + formatDate.getMonth()+1).slice(-2)}/${formatDate.getFullYear()}`
-    const tformat = `${formatDate.getHours()}:${formatDate.getMinutes()}:${formatDate.getSeconds()}`
-    const timeStamp = dformat + ' ' + tformat
+    const formatDate = new Date();
+    const dformat = formatDate.toDateString();
+    const tformat = formatDate.toLocaleTimeString();
+    const timeStamp = dformat + ' ' + tformat;
```

Comment:
Let's prefer string interpolation
```suggestion
    const timeStamp = `${dformat} ${tformat}`;
```